src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -21,13 +21,15 @@ public static class ActivityExtensions
 {
     /// <summary>
     /// Sets the status of activity execution.
-    /// Activity class in .NET does not support 'Status'.
-    /// This extension provides a workaround to store Status as special tags with key name of otel.status_code and otel.status_description.
-    /// Read more about SetStatus here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status.
     /// </summary>
+    /// <remarks>
+    /// This method is obsolete.
```

Comment:
can you give the alternate API to use right here, so users dont have to open the link and read about it? 
(The content from changelog.md file is good to be put here.)

Reply:
Done